components/sysutils/clamav/Makefile

```diff
@@ -117,6 +119,16 @@ COMPONENT_TEST_TRANSFORMS += \
 REQUIRED_PACKAGES += system/library/iconv/unicode
 REQUIRED_PACKAGES += developer/check
 
+# currently there is no system/library/g++-4-runtime installed only two links are needed
+# to build the package:
+#
+#       ln -s $(GCC_LIBDIR)/libgcc_s.so.1 /usr/lib/$(MACH64)/libgcc_s.so.1
+#       ln -s $(GCC_LIBDIR)/liblibssp.so.0 /usr/lib/$(MACH64)/libssp.so.0
+#
+# Also some other package can provide this two links (which one?)
+# Basically it proves that the very old gcc is not needed as also mentioned by Joshua
+# M. Clulow
+
```

Comment:
Using these links into gcc-7 doesn't quite work as expected:

Test project /export/home/andreas/oi-userland/components/sysutils/clamav/build/amd64
    Start 1: libclamav
1/6 Test #1: libclamav ........................   Passed   18.57 sec
    Start 2: libclamav_rust
2/6 Test #2: libclamav_rust ...................   Passed    0.44 sec
    Start 3: clamscan
3/6 Test #3: clamscan .........................   Passed    9.67 sec
    Start 4: clamd
4/6 Test #4: clamd ............................***Failed  132.11 sec
    Start 5: freshclam
5/6 Test #5: freshclam ........................   Passed    4.70 sec
    Start 6: sigtool
6/6 Test #6: sigtool ..........................   Passed    2.89 sec

83% tests passed, 1 tests failed out of 6

Total Test time (real) = 168.40 sec

The following tests FAILED:
	  4 - clamd (Failed)
Errors while running CTest

This failure is repeatable. I will run more tests with gcc-10 and gcc-11.